### ISSUE TITLE
FrameNet corpus reader should sort multiword lexemes on "order" attribute

### ISSUE THREAD

**Issue by nschneid** [nltk/nltk#1132]
It looks like the ordering of a lexical unit's lexemes in the XML does not always follow the "order" attribute:

``` py
>>> fn.lus('write down.v')[0].lexemes
[{'POS': 'ADV', 'breakBefore': 'true', 'headword': 'false', 'name': 'down', 'order': 2}, 
{'POS': 'V', 'breakBefore': 'false', 'headword': 'true', 'name': 'write', 'order': 1}]
```

These should be sorted on loading so "write" comes first.

(N.B. This example is using FN 1.6, but the XML schema has not changed.)

**Comment by nschneid** [comment]
I can fix this. Making a note so I don't forget.

**Comment by nschneid** [comment]
Relatedly: there are a few number LUs whose lexeme name is loaded as an int rather than a string:

``` py
('1000.num', {'POS': 'NUM', 'breakBefore': 'false', 'headword': 'true', 'name': 1000, 'order': 1})
('1200.num', {'POS': 'NUM', 'breakBefore': 'false', 'headword': 'true', 'name': 1200, 'order': 1})
('450.num', {'POS': 'NUM', 'breakBefore': 'false', 'headword': 'true', 'name': 450, 'order': 1})
('400.num', {'POS': 'NUM', 'breakBefore': 'false', 'headword': 'true', 'name': 400, 'order': 1})
('6500.num', {'POS': 'NUM', 'breakBefore': 'false', 'headword': 'true', 'name': 6500, 'order': 1})
```

**Comment by stevenbird** [comment]
@nschneid – If you accept the invitation to the nltk team I can assign this to you.

**Comment by nschneid** [comment]
Fixed when #1556 was merged.
